test/lib/commands/link.js

```diff
@@ -369,6 +369,42 @@ t.test('link pkg already in global space when prefix is a symlink', async t => {
   t.matchSnapshot(await printLinks(), 'should create a local symlink to global pkg')
 })
 
+t.test('saving link to package file', async t => {
```

Comment:
Where is `save` being set to `false` in this test?

Reply:
done